README.md

```diff
@@ -1,108 +1,112 @@
+
+### **Requirements**
+You need at least one server or virtual machine on which Hopsworks will be installed with at least the following specification:
+- Centos/RHEL 7.x or Ubuntu 18.04;
+- at least 32GB RAM,
+- at least 8 CPUs,
+- 100 GB of free hard-disk space,
+- outside Internet access (if this server is air-gapped, contact us for support),
+- a UNIX user account with sudo privileges.
+<br />
 
 <a name="docs"></a>
-## Docs
-[Hopsworks documentation](https://hopsworks.readthedocs.io/en/latest/overview/overview.html#) includes a user-guide, Feature Store documentation, and an Administrator Guide. There is also dedicated [documentation for the Hopsworks Feature Store](https://docs.hopsworks.ai/feature-store-api/latest/).
-
-Hopsworks REST API is documented with Swagger and hosted by SwaggerHub.
-- **hopsworks-api** - [https://app.swaggerhub.com/apis-docs/logicalclocks/hopsworks-api](https://app.swaggerhub.com/apis-docs/logicalclocks/hopsworks-api)
-- **hopsworks-ca** - [https://app.swaggerhub.com/apis-docs/logicalclocks/hopsworks-ca](https://app.swaggerhub.com/apis-docs/logicalclocks/hopsworks-ca)
-
-To build and deploy swagger on your own Hopsworks instance you can follow the instructions found in 
-[this](https://hopsworks.readthedocs.io/en/stable/user_guide/hopsworks/microservices.html?highlight=swagger#swagger-ui) guide.
-
-
-<a name="who"></a>
-## Who’s behind Hopsworks?
-Hopsworks started as an open-source collaborative project at [KTH University](https://www.kth.se/en), [RISE](https://www.ri.se/en), and has more recently been taken on by [Logical Clocks](https://www.logicalclocks.com/). Several funding bodies have helped contribute to its development including: [European Commission (FP7, H2020)](https://ec.europa.eu/), [EIT](https://eit.europa.eu/), [SSF](https://strategiska.se/), [Vinnova](https://www.vinnova.se/) and [Celtic-Next](https://www.celticnext.eu/).
+# 🎓 Documentation and API
+## **Documentation**
+[Hopsworks documentation](https://docs.hopsworks.ai) includes user-guides, Feature Store documentation, and an Administrator Guide. We also include concepts to help user navigates the abstractions and logics of the feature stores and MLOps in general:
+- **Feature Store:** [https://docs.hopsworks.ai/3.0/concepts/fs/](https://docs.hopsworks.ai/3.0/concepts/fs/)
+- **Projects:** [https://docs.hopsworks.ai/3.0/concepts/projects/governance/](https://docs.hopsworks.ai/3.0/concepts/projects/governance/)
+- **MLOps:** [https://docs.hopsworks.ai/3.0/concepts/mlops/prediction_services/](https://docs.hopsworks.ai/3.0/concepts/mlops/prediction_services/)
+
+## **APIs**
+Hopsworks API documentation is divided in 3 categories; Hopsworks API covers project level APIs, Feature Store API covers feature groups, training datasets and connectors and finally MLOPs API covers Model Registry, Serving and Deployment. 
```

Comment:
covers feature groups, feature views and connectors. also type mlops with consistent capitalization.